src/libcsg/modules/io/xyzreader.cc

```diff
@@ -18,8 +18,7 @@
 #include <vector>
 #include <boost/lexical_cast.hpp>
 #include <votca/tools/getline.h>
-#include "xyzreader.h"
-
+#include <votca/csg/xyzreader.h>
```

Comment:
I think that local includes are always preferred actually.

Reply:
okay, but it is in a different directory now, so I had to change the includes

Reply:
ok